test/functional/apps/discover/_discover.js

```diff
@@ -260,13 +260,14 @@ export default function ({ getService, getPageObjects }) {
     });
 
     describe('invalid time range in URL', function () {
-      it('should display a "Invalid time range toast"', async function () {
+      it('should get the default timerange', async function () {
         await PageObjects.common.navigateToUrl('discover', '#/?_g=(time:(from:now-15m,to:null))', {
           useActualUrl: true,
         });
```

Comment:
Might be failing because there was a different default time range configured in Line250

Reply:
Yeah - just found it too!